tools/submission/submission-checker.py

```diff
@@ -166,8 +166,9 @@
 }
 
 VALID_DIVISIONS = ["open", "closed"]
-REQUIRED_PERF_FILES = ["mlperf_log_accuracy.json", "mlperf_log_summary.txt", "mlperf_log_detail.txt"]
-REQUIRED_ACC_FILES = REQUIRED_PERF_FILES + ["accuracy.txt"]
+REQUIRED_PERF_FILES = ["mlperf_log_summary.txt", "mlperf_log_detail.txt"]
+OPTIONAL_PERF_FILES = ["mlperf_log_accuracy.json"]
```

Comment:
I think it would be cleaner to say:
```
OPTIONAL_PERF_FILES = ["mlperf_log_accuracy.json", "mlperf_log_trace.json" ]
OPTIONAL_ACC_FILES = ["mlperf_log_trace.json"]
```
At the moment, `"mlperf_log_trace.json"` is generated but is listed neither as optional nor as required.

Reply:
changed it.